### PR TITLE
updating parametric code to be consistent with notes

### PR DESCRIPTION
**Pull request by srobertp**
@lwy5515 What do you think?  Should we merge this in now?  I changed a few things and added a script to generate error analysis tables for us, as well as a function version of the main script.  If you are okay with it, we should go ahead and merge it.

### CODE REVIEW

**Comment by srobertp** [comment]
Here is the error table for the new code:

<pre>
 n    error   rate
  8 4.775e-01 0.00
 16 1.849e-01 1.37
 32 5.516e-02 1.75
 64 1.441e-02 1.94
128 3.644e-03 1.98
256 9.134e-04 2.00
</pre>

**Comment by lwy5515** [comment]
Let me merge it right now

File: torus_parametric/local_assembling.m [hunk 1]
```diff
@@ -1,116 +1,228 @@
...
+%
+%    = ... simplifications described below
+%
+%    = \int_{T}  (alpha*\nabla\phistar_j^T inv(G_Gamma) \nabla\phistar_i
+%                  + beta*phistar_j*phistar_i  )
+%                  * Q_Gamma dy
+%
+% where \nabla_S is the surface gradient on surface X0(T).
+%
+% The right hand side is defined on X0(T) so as to be consistent
```
Comment (srobertp) [inline torus_parametric/local_assembling.m:23]: testing comment in code.  I am on line 23 of local_assembling

Comment (srobertp) [inline torus_parametric/local_assembling.m:23]: another note on the line

**Pull request merged**